src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/SubjectAccessRequestService.kt

```diff
@@ -35,7 +35,7 @@ class SubjectAccessRequestService(
       inductionService.getInductionForPrisoner(prisonNumber)
         .takeIf { fromDateInstance == null || it.createdAt?.isAfter(fromDateInstance) ?: true }
         .takeIf { toDateInstance == null || it?.createdAt?.isBefore(toDateInstance) ?: true }
-    } catch (e: InductionNotFoundException) {
+    } catch (e: uk.gov.justice.digital.hmpps.domain.learningandworkprogress.induction.InductionNotFoundException) {
```

Comment:
Is the full class path needed here?

Reply:
Blimey, good spot! 😁 
I thought I'd caught and reverted all of those! Its basically where the IDE did the refactoring and for some reason wanted to do it in steps (I think) - for some reason it fully qualified all of the exceptions like this, and then the linter moaned at me because there was an unused import!

I'll correct it 👍